### ISSUE TITLE
playSegments: Cannot read property '0' of undefined

### ISSUE THREAD

**Issue by jadbox** [airbnb/lottie-web#1322]
Version: "react-lottie": "^1.2.3",

Calling playSegments seems to always yield the below error.

Tried all of these options and got the same error:
table.playSegments({0:[0, 40]}, true);
table.playSegments([0, 40], true);
table.playSegments([[0, 40]], true);

Error:
```
lottie.js:9250 Uncaught TypeError: Cannot read property '0' of undefined
    at AnimationItem.playSegments (lottie.js:9250)
    at Lottie.playSegments (index.js:151)
```

Options:
```
const tableOptions = {
  loop: true,
  autoplay: true,
  path: 'assets/table.json',
  subframeEnabled: false,
  rendererSettings: {
    preserveAspectRatio: 'xMidYMid slice'
  }
};
```

React placement:
```
<Lottie
  options={tableOptions}
  ref={this.tableEl}
/>
```

**Comment by bodymovin** [comment]
hi, can you share a link with the issue that I can check?

**Comment by jadbox** [comment]
I'll need to check if I can share the exact code, but here's an image of the call site stack:
<img width="652" alt="2018-10-23_1649" src="https://user-images.githubusercontent.com/595843/47397330-a3461700-d6e3-11e8-95df-b1fa616735b0.png">

**Comment by bodymovin** [comment]
I don't have the full context, but it seems that react-lottie is expecting to get the segments as a prop, and not as an argument of the method playSegments.
That's the reason why even if you're passing the segments they are not being picked up.

**Comment by jadbox** [comment]
@bodymovin I see, but why is the playSegments function ignoring function parameters and opting to pull from `this.props`? Does it have something to do with accessing a Lottie component using a React.createRef() reference? (see the `const t= this.tableEl.current`). I've also tried setting the segments as a property (see below) and then calling `t.playSegments()` but the clip plays all the frames and then plays the segment that I passed as a prop.
